MathlibTest/norm_num_rpow.lean

```diff
@@ -13,3 +13,24 @@ example : (-1/3 : ℝ) ^ (-3 : ℝ) = -27 := by norm_num1
 example : (1/2 : ℝ) ^ (-3 : ℝ) = 8 := by norm_num1
 example : (2 : ℝ) ^ (-3 : ℝ) = 1/8 := by norm_num1
 example : (-2 : ℝ) ^ (-3 : ℝ) = -1/8 := by norm_num1
+
+
```

Comment:
nit, just one blank line:
```suggestion
```

unless there's a separation you're trying to emphasize